tempus_fugit_minecraft/main.py

```diff
@@ -681,6 +717,12 @@ def on_mouse_press(self, x, y, button, modifiers):
             mouse button was clicked.
 
         """
+        if self.paused:
+            if self.within_label(x, y, self.resume_label):
+                self.resume_game()
+            elif self.within_label(x, y, self.quit_label):
+                self.close()
+
```

Comment:
I think after handling this, we should return from the function. If we didn't, and we were to resume, the click on the resume_label would also process to the block underneath it. 
```py
        if self.paused:
            if self.within_label(x, y, self.resume_label):
                self.resume_game()
            elif self.within_label(x, y, self.quit_label):
                self.close()
            return
```

Reply:
I'm not sure this change does anything as there is not a code block underneath it to run.